stack.yaml

```diff
@@ -0,0 +1,6 @@
+flags: {}
+packages:
+- '.'
+extra-deps:
+- cabal-helper-0.5.1.0
```

Comment:
Exactly, it's not part of the `lts-3.0` resolver, see [here](https://github.com/commercialhaskell/stack/wiki/stack.yaml#extra-deps).
You can also automatically have extra-deps calculated and added by using `stack solver --modify-stack-yaml`.